test/integration/targets/git/tasks/checkout-new-tag.yml

```diff
@@ -48,6 +48,9 @@
     that:
       - not update_new_tag|changed
       - "'newtag' in listoftags.stdout_lines"
+  # This test fails on osx.
```

Comment:
It doesn't appear to be git version related. It happens on git 2.7.4 on OS X, but not on the same git version on Ubuntu. Upgrading git to 2.10.1 still has the same problem.